src/mscorlib/src/System/Reflection/RuntimeConstructorInfo.cs

```diff
@@ -462,18 +461,5 @@ public override Object Invoke(BindingFlags invokeAttr, Binder binder, Object[] p
             return RuntimeMethodHandle.InvokeMethod(null, null, sig, true);
         }
         #endregion
-
-        #region ISerializable Implementation
-        public void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            throw new PlatformNotSupportedException();
```

Comment:
How would you call GetObjectData() on a ConstructorInfo? The method doesn't exist anywhere in the contract type hierarchy (or even in the implementation stack for that matter.)

GetObjectData() only exists on Module and Assembly, and the Runtime classes for those do override with PNSE.